h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostMojoWriter.java

```diff
@@ -28,5 +29,6 @@ protected void writeModelData() throws IOException {
     writekv("cat_offsets", model._output._catOffsets);
     writekv("use_all_factor_levels", model._output._useAllFactorLevels);
     writekv("sparse", model._output._sparse);
+    writeblob("feature_map", model.model_info().getFeatureMap().getBytes(Charset.forName("UTF-8")));
```

Comment:
Minor: We could save the UTF-8 charset in a static variable & not obtain it every time.